README.md

```diff
@@ -1,23 +1,44 @@
-<!--
-Avoid using this README file for information that is maintained or published elsewhere, e.g.:
+# Identity Platform Login UI Operator
 
-* metadata.yaml > published on Charmhub
-* documentation > published on (or linked to from) Charmhub
-* detailed contribution guide > documentation or CONTRIBUTING.md
+## Description
 
-Use links instead.
--->
+This repository hosts the Kubernetes Python Operator for Identity Platform Login UI.
+For more details, visit https://github.com/canonical/identity-platform-login-ui
 
-# kratos-ui
+## Usage
 
-This is a UI for the Ory Kratos identity server. It was based on the [kratos-selfservice-ui-react-nextjs](https://github.com/ory/kratos-selfservice-ui-react-nextjs/).
+The Identity Platform Login UI Operator may be deployed using with the following commands:
 
-## Other resources
+```console
+git clone https://github.com/canonical/identity-platform-login-ui-operator
+cd identity-platform-login-ui-operator
+charmcraft pack
+juju deploy ./identity-platform-login-ui_ubuntu-*.charm --resource oci-image=$(yq eval '.resources.login-ui-image.upstream-source' metadata.yaml)
```

Comment:
We should point to the charmhub version of the charm in the README, not a locally built one

Reply:
changed to use future charmhub version